.github/workflows/ci.yml

```diff
@@ -21,7 +21,9 @@ jobs:
       - name: Linting
         run: |
           cd rrule
-          cargo clippy -- -D warnings
+          # revert it back to the following line, the deprecated functions are removed from chrono
+          # cargo clippy -- -D warnings
+          cargo clippy
```

Comment:
The problem is chrono-tz deprecated them, but chrono is using the deprecated apis.
So let's say they are not in sync atm.

So it's not a forgotten 🙈

Reply:
I am not sure I am following. Running `cargo clippy` locally complains about using the deprecated `chrono::Date` type. How does that relate to `chrono-tz`?

Reply:
Here is our deprecated impl:
![Screenshot from 2023-03-25 08-55-02](https://user-images.githubusercontent.com/45714/227705095-202b5d0d-d3ca-4cbf-856e-957c6266fc44.png)

Now when I go to `chrono`'s `Local` impl, you see they use a deprecated method also (My IDE didn't mark it as deprecared, but it is the one!)
And the whole trait method is also marked as deprecated.
![Screenshot from 2023-03-25 08-55-42](https://user-images.githubusercontent.com/45714/227705129-f7f9e006-b798-444c-b0e2-323098c2b7a7.png)


AFAICS, this `impl TimeZone for` we have, is kinda combination of its impl for chrono's `Local` and chorono-tz's `Tz`, which you can find them [here](https://github.com/chronotope/chrono/blob/main/src/offset/local/mod.rs#L113) and [here](https://github.com/chronotope/chrono-tz/blob/main/chrono-tz/src/timezone_impl.rs#L343).

In the second one, you can see how they solved it, but I think generally spending time on something deprecated which will be removed soon from the main `trait` is useless.

Reply:
Thanks for the ping @omid , been a bit swamped with other things lately. I think we are good to merge this soon. Since you have the most context on the deprecation issue, do you mind creating a small GH issue here with potential follow up work and which API improvements from chrono / chrono-tz we are waiting for?

Reply:
sure: https://github.com/fmeringdal/rust-rrule/issues/91